index.html

```diff
@@ -4,9 +4,114 @@
+        </header>
+        <div class="main-page">
+            <div class="menu">
+                <a href="./index.html">Home</a>
+                <a href="./about.html">About</a>
+            </div>
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <div class="main-container">
+                <h1>"Hello! IT Lovers"</h1>
+                <h2>IT live annual confrence 2022</h2>
+                <p class="main-desc">This confrence is hosted by the major tech companies in Iraq and it'll feature's many of the lead indivsuals of the tech indusry</p>
+                <div class="date">
+                    <h3> 2022.11.31(THU) ~ 2022.12.1(FRI)</h3>
+                    <p>@ National Theater, Baghdad, Iraq</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="main-program">
+        <div class="main-program-container">
+            <h3>Main Program</h3>
+            <hr>
+            <div class="main-program-content">
+                <ul class="main-grid">
+                    <li class="grid-item">
+                        <img src="./assets/lecturer.png" alt="">
+                        <h3>Lecture</h3>
+                        <p>Listen to the speakers from various companies about thier experince</p>
+                    </li>
+                    <li class="grid-item">
+                        <img src="./assets/exhibition.png" alt="">
+                        <h3>Exhibition</h3>
+                        <p>Appreciate various exhibitions by diffrent companies and products</p>
+                    </li>
+                    <li class="grid-item">
+                        <img src="./assets/forum.png" alt="">
+                        <h3>Forum</h3>
+                        <p>Have the time to share your thoughts and opinoin with experts about difrrent topics</p>
+                    </li>
+                    <li class="grid-item">
+                        <img src="./assets/workshop.png" alt="">
+                        <h3>Workshop</h3>
+                        <p>Participate in diffrent workshop from provided by senior engineers</p>
+                    </li>
+                    <li class="grid-item">
+                        <img src="./assets/Networking.png" alt="">
+                        <h3>Networking</h3>
+                        <p>Get opportunities to network with affilaates around the countery also after the conference</p>
+                    </li>
+                </ul>
+            </div>
+            <button type="button">Join IT Live Annual Conference</button>
+            <h3 id="h3">See The Whole Program</h3>
+        </div>
+    </section>
+    <section class="featured-speakers">
+        <h2>Featured Speakers</h2>
+        <hr>
+        <div class="speakers">
+        </div>
+    </section>
+    <section class="footer">
```

Comment:
- [ ] You did a great job working on your footer section! 💯 However, HTML includes a `<footer>` tag to provide better semantics to this. Please consider changing this `<section>` to `<footer>`! 🤓 🛠️